src/app/client/src/app/modules/dial-code-search/services/dial-code/dial-code.service.ts

```diff
@@ -1,5 +1,6 @@
-import { ConfigService } from '@sunbird/shared';
-import { SearchService, PlayerService } from '@sunbird/core';
+import { ConfigService, ServerResponse} from '@sunbird/shared';
+import { SearchService, PlayerService, PublicDataService} from '@sunbird/core';
+import { UserService } from '../../../../modules/core/services/user/user.service';
```

Comment:
add UserService in above line like
import { SearchService, PlayerService, PublicDataService} from '@sunbird/core';